allure-generator/src/main/javascript/utils/settingsFactory.js

```diff
@@ -35,7 +34,7 @@ const widgetGridPluginDefaults = {
 function getGlobalSettings() {
   const SettingsModel = LocalStorageModel.extend({
     defaults() {
-      return globalSettingsDefaults;
+      return { ...globalSettingsDefaults, language: document.documentElement.lang || "en" };
```

Comment:
I have a question.

Would you consider adding an option like `auto` for language?

This can work in a similar way to this:

https://github.com/sonic16x/interslavic/pull/222/files#diff-6c9b6c5e0990d5094cdffba8ac89a46c924ab7f2a69ba718d819c95118e3ee77

The idea is to fall back to English when there's no good match with `navigator.languages`. However, if there is a match, you can set the language to the user's most preferred one.

Reply:
As far as I remember, we used user-preferred language before, and the community asked us to use English by default instead. It may make sense after this PR is merged (so "auto" by default, by you can change it to whatever language you want by using the generator option ), although I don't have such a feature request in my backlog at the moment.

Reply:
> so "auto" by default, by you can change it to whatever language you want by using the generator option

I would even suggest keeping "en" as the default language, unless "auto" is specifically configured. 😄

Regardless, this feature is very useful. I have been wondering if it is possible to set a default language ever since I added Interslavic translations to the project. Good to see things moving. 👍